src/citrine/ara/variables.py

```diff
@@ -588,7 +588,7 @@ class IngredientQuantityInOutput(Serializable['IngredientQuantityInOutput'], Var
     ingredient_name = properties.String('ingredient_name')
     quantity_dimension = properties.Enumeration(IngredientQuantityDimension, 'quantity_dimension')
     process_templates = properties.List(properties.Object(LinkByUID), 'process_templates')
-    typ = properties.String('type', default="ing_id_in_output", deserializable=False)
+    typ = properties.String('type', default="ing_quantity_in_output", deserializable=False)
```

Comment:
Drive-by bug, right?  Is worth breaking it out to figure out why it didn't have test coverage?

Reply:
hmm I think I can probably add a serde test that would have detected this issue. Let me see.

Reply:
https://github.com/CitrineInformatics/citrine-python/blob/3ea5cfb5324b4623e0e8953c78c4234f9dccf84c/tests/ara/test_variables.py#L35 was actually the test I was planning to write, but it wasn't catching the issue because the one type's field's was a subset of the other and the `Variable.__eq__` method had different behavior depending on which variable is on the left/right. https://github.com/CitrineInformatics/citrine-python/blob/1dcc8c10013c127a5df8287bfd4e7402f192b753/src/citrine/ara/variables.py#L42

Reply:
The new `__eq__` method in my latest commit also checks types, which caused a test failure without the type hint update in this PR.

Reply:
Excellent.